launch/buoy_task.launch

```diff
@@ -3,7 +3,7 @@
     <node
 	    name="ign_gazebo" pkg="ros_ign_gazebo" type="ign_gazebo" output="screen" required="true"
 	    args="$(find auv-sim)/worlds/buoy_task.sdf">
-        <env name="IGN_GAZEBO_RESOURCE_PATH" value="$(env HOME)/AUV-2023/catkin_ws/src/auv-sim"/>
+        <env name="IGN_GAZEBO_RESOURCE_PATH" value="$(env HOME)/AUV-2023/catkin_ws/src/sim/auv-sim"/>
```

Comment:
why is the path different now?

Reply:
we changed the file structure to have our own ros ign bridge with custom queue size